test/api/campuses_test.rb

```diff
@@ -37,6 +45,15 @@ def test_post_campuses
     assert_equal 0, campus[:mode]
   end
 
+  def test_student_post_campuses
+    user_student = FactoryGirl.create(:user, :student)
+    data_to_post = { campus: FactoryGirl.build(:campus, mode: 'timetable') }
+
+    post_json  with_auth_token("/api/campuses", user_student), data_to_post
+    assert_equal 403, last_response.status
```

Comment:
Add checks that the created campus matches the data you sent through

Reply:
Sorry for the confusion. This method is designed to test that students lack auth to put data.
I've updated the method name to reflect this more clearly.